libr/core/cmd.c

```diff
@@ -5000,7 +5008,7 @@ DEFINE_HANDLE_TS_FCN(arged_command) {
 		char *exec_string = r_str_newf (".%s", cmd_str);
 		free (cmd_str);
 		free (command_str);
-		bool res = r_core_cmd (state->core, exec_string, false) != -1;
+		res = r_core_cmd (state->core, exec_string, false) != -1;
```

Comment:
r_core_cmd returns the {0, -1, -2, true, false} thing. Assigning to RCoreCmdStatus might cause problems.